core/assignment.go

```diff
@@ -234,6 +235,10 @@ func (c *StdAssignmentCoordinator) CalculateChunkLength(state *OperatorState, bl
 
 	for {
 
+		if chunkLength > blobLength {
```

Comment:
would it make sense to place this check as a condition in `ValidateChunkLength`?

Reply:
I don't think it's essential, but it probably makes sense. I'll add it.

Reply:
oh I was thinking if we moved this to `ValidateChunkLength`, we wouldn't even need to validate it here as its done in L256

Reply:
Makes sense. Changed it.